images/hail-gcp-notebook/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM gcr.io/deeplearning-platform-release/base-cpu
+
+RUN apt-get update && \
```

Comment:
```suggestion
# ignore linting errors for pinning apt-get and pip packages
# hadolint ignore= DL3008, DL3013
RUN apt-get update && \
```

Reply:
Actually I did pin those packages, just forgot to add that commit. I think it's a good idea, having been bitten by version updates a number of times recently.

Reply:
Reverted on this, after discussing on Slack. It might be handy to be able to update this image easily without requiring a commit.